views/js/review/runner.js

```diff
@@ -0,0 +1,221 @@
+                    category: 'communicator'
+                },
+                plugins: defaultPlugins
+            },
+            options: {
+                readOnly: config.readOnly,
+                fullPage: config.fullPage,
+            }
+        };
+
+        const testData = {
+            config: {
+                allowShortcuts: true,
+                shortcuts: {
+                    'next-prev-review': {
+                        next: 'arrowright',
+                        prev: 'arrowleft'
+                    }
+                },
+            }
+        };
+
+        const items = [{
+            itemDefinition: 'item-1',
+            state: null,
+        }, {
+            itemDefinition: 'item-2',
+            state: {
+                RESPONSE: {
+                    response: {
+                        base: {
+                            integer: 0
+                        }
+                    }
+                },
+                RESPONSE_1: {
+                    response: {
+                        list: {
+                            identifier: [
+                                "choice_3",
+                                "choice_4",
+                                "choice_5",
+                            ]
+                        }
+                    }
+                },
+                RESPONSE_3: {
+                    response: {
+                        base: {
+                            integer: 327000
+                        }
+                    }
+                },
+            }
+        }, {
+            itemDefinition: 'item-3',
+            state: {
+                RESPONSE: {
+                    response: {
+                        list: {
+                            "identifier":["choice_2"]
+                        }
+                    }
+                },
+                "RESPONSE_1":{
+                    "response":{
+                        "list":{
+                            "pair":[
+                                ["choice_5","choice_6"],
+                                ["choice_4","choice_7"]
+                            ]
+                        }
+                    }
+                }
+            },
+        }];
+
+        const testContex = {
+            itemIdentifier: "item-1",
+            itemPosition: 0,
+            isLinear: false,
+            canMoveBackward: true
+        };
+
+        return runnerComponentFactory(container, testRunnerConfig, template || runnerTpl)
+            .on('render', function() {
+                const {fullPage, readOnly} = this.getConfig().options;
+                this.setState('fullpage', fullPage);
+                this.setState('readonly', readOnly);
+
+                // start - temporarily for mock-data
+                const runner = this.getRunner();
+                runner.setTestMap(testmap);
+                runner.setTestContext(testContex);
+                runner.setTestData(testData);
```

Comment:
Not the best place. It should be set by the provider, on init. And supplied through the proxy.